var/spack/repos/builtin/packages/py-deepsig-biocomp/package.py

```diff
@@ -19,11 +19,11 @@ class PyDeepsigBiocomp(PythonPackage):
 
     version("1.2.5", sha256="e954b815d63c221c564c7d3fe27123d7cd2c39b191d6107369ab095d506496e0")
 
-    depends_on("python@3.8", type=("build", "run"))
+    depends_on("python@3.8:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     depends_on("py-biopython@1.78:", type=("build", "run"))
-    depends_on("py-keras@2.4.3", type=("build", "run"))
-    depends_on("py-tensorflow@2.2.0", type=("build", "run"))
+    depends_on("py-keras@2.4.3:", type=("build", "run"))
+    depends_on("py-tensorflow@2.2.0:", type=("build", "run"))
```

Comment:
Is use of the `:` in these three packages are not consistent with `requirements.txt`'s:

```
biopython==1.78
Keras==2.4.3
tensorflow==2.2.0
```

Yet, `setup.py` supports lines 24 and 25 but no constraint on line 26:

```
    install_requires=[
        'biopython >= 1.78',
        'Keras >= 2.4.3',
        'tensorflow'
    ],
```